sdl_viewer/src/box_drawer.rs

```diff
@@ -164,13 +164,13 @@ impl BoxDrawer {
     // Then we use 'world_to_gl' to transform it into clip space.
     pub fn draw_outlines(
         &self,
-        cuboid: &Aabb3<f64>,
+        cuboid: &Aabb<f64>,
         world_to_gl: &Matrix4<f64>,
         color: &color::Color<f32>,
     ) {
-        let dim = cuboid.dim() / 2.0;
-        let scale_matrix = Matrix4::from_nonuniform_scale(dim.x, dim.y, dim.z);
-        let translation_matrix = Matrix4::from_translation(cuboid.center().to_vec());
+        let dim = cuboid.max() - cuboid.center();
```

Comment:
Do
```suggestion
        let dim = cuboid.dim() / 2.0;
```
if you decide to implement `Aabb::dim()`

Reply:
Done